PersonApi/V1/Domain/ApiLink.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace PersonApi.V1.Domain
+{
+    public class ApiLink
```

Comment:
I think this class should not be part of the domain. I guess a better place  should be inside the API response. This can be an interim solution. A better approach should be to define inside a NuGet package a BaseResponseObject which can be inherited by all the API responses classes.

Reply:
I agree that (a) base response class(es) in a separate package would be useful.
In this case though, I an happy for this (ApiLInk) class to just be in the response, but if it is the case then where will the information it contains be held in the model? 
Presumably this information will be added to a person from the processing of an event raised by another service, but something will need to be on this (or another) model in some form.

Reply:
Updated